proto/asgt/v2/type/training.proto

```diff
@@ -27,4 +27,6 @@ message Training {
     google.protobuf.Timestamp finish_time = 5;
     // Specifies dataset the training ran on.
     asgt.type.Dataset dataset = 6;
+    // Specifies when is the training scheduled.
+    google.protobuf.Timestamp schedule_time = 7;
 }
```

Comment:
I noticed we forgot to update the comments here last time we made changes into the service 😄 So we can just do it now. Above the `training_status` field, where it explains the `SCHEDULED` status, it says that the training could happen between 1 minute and 6 hours. However, this is now between 1 minute and 24 hours - we changed this to save costs on compute. Also, there's a word missing in the sentence `PENDING - The training is ready to be picked up a worker.` - it's missing a "by".